core/src/trezor/ui/layouts/tt_v2/__init__.py

```diff
@@ -996,45 +1001,93 @@ async def confirm_modify_output(
     amount_change: str,
     amount_new: str,
 ) -> None:
-    await raise_if_not_confirmed(
-        interact(
-            ctx,
+    send_button_request = True
+    while True:
+        if send_button_request:
+            await button_request(
+                ctx,
+                "modify_output",
+                ButtonRequestType.ConfirmOutput,
+            )
+        await raise_if_not_confirmed(
+            ctx.wait(
+                RustLayout(
+                    trezorui2.confirm_blob(
+                        title="MODIFY AMOUNT",
+                        data=address,
+                        verb="CONTINUE",
+                        verb_cancel=None,
+                        description="Address:",
+                        extra=None,
+                    )
+                )
+            )
+        )
+
+        if send_button_request:
+            send_button_request = False
+            await button_request(
+                ctx,
+                "modify_output",
+                ButtonRequestType.ConfirmOutput,
+            )
+        result = await ctx.wait(
             RustLayout(
                 trezorui2.confirm_modify_output(
-                    address=address,
                     sign=sign,
                     amount_change=amount_change,
                     amount_new=amount_new,
                 )
             ),
-            "modify_output",
-            ButtonRequestType.ConfirmOutput,
         )
-    )
+
+        if result is CONFIRMED:
+            break
 
 
 async def confirm_modify_fee(
     ctx: GenericContext,
+    title: str,
     sign: int,
     user_fee_change: str,
     total_fee_new: str,
     fee_rate_amount: str | None = None,
 ) -> None:
-    await raise_if_not_confirmed(
-        interact(
-            ctx,
-            RustLayout(
-                trezorui2.confirm_modify_fee(
-                    sign=sign,
-                    user_fee_change=user_fee_change,
-                    total_fee_new=total_fee_new,
-                    fee_rate_amount=fee_rate_amount,
-                )
-            ),
-            "modify_fee",
-            ButtonRequestType.SignTx,
+    layout = RustLayout(
+        trezorui2.confirm_modify_fee(
+            title=title.upper(),
+            sign=sign,
+            user_fee_change=user_fee_change,
+            total_fee_new=total_fee_new,
         )
     )
+    await button_request(
+        ctx,
+        "modify_fee",
+        ButtonRequestType.SignTx,
+    )
+
+    while True:
+        result = await ctx.wait(layout)
+
+        if result is CONFIRMED:
+            return
+        elif result is INFO:
```

Comment:
There seem to be no tests for this `INFO` button

Reply:
Fixed in [c0cfca8](https://github.com/trezor/trezor-firmware/pull/3029/commits/c0cfca8a39ed8bb5da2959741cb3dbdf11deab29).